### ISSUE TITLE
What lr schedule to use to finetune T5 model using AdamOptimizer

### ISSUE THREAD

**Issue by LiweiPeng** [google-research/text-to-text-transfer-transformer#230]
The T5 model was finetuned using AdaFactor optimizer with constant learning rate value. The Google [Memory-Efficient Adaptive Optimization paper](https://arxiv.org/pdf/1901.11150v2.pdf) mentioned that Adam Optimizer is a little bit better in converge than AdaFactor though it consumes much more memory.

Has someone tried AdamOptimizer for finetuning T5 model? If so, what's the recommended learning rate schedule (const, linear, polynomial, etc)? Thanks.

**Comment by craffel** [comment]
I played around with Adam when working on the Hugging Face Transformers model since Transformers doesn't have AdaFactor implemented. Using Adam with a constant learning rate of 1e-4 seemed to work about the same as our base fine-tuning setup (AdaFactor with constant LR of 0.001). I'd suggest playing around with it a bit. I don't really have patience for tweaking the hyperparameters of learning rate schedules so I'm always inclined to just try a constant LR. Let us know if you find something that works better.

**Comment by craffel** [comment]
Oh and to be more specific, this was the AdamW optimizer: https://github.com/google-research/text-to-text-transfer-transformer/blob/master/t5/models/hf_model.py#L55
